src/mumble/MainWindow.h

```diff
@@ -70,8 +70,8 @@ class MainWindow : public QMainWindow, public MessageHandler, public Ui::MainWin
 		QMenu *qmChannel;
 		QMenu *qmDeveloper;
 		QMenu *qmTray;
-		QIcon qiIcon, qiIconMutePushToMute, qiIconMuteSelf, qiIconMuteServer, qiIconDeafSelf, qiIconDeafServer, qiIconMuteSuppressed;
-		QIcon qiTalkingOn, qiTalkingWhisper, qiTalkingShout, qiTalkingOff;
+		QPixmap qiIcon, qiIconMutePushToMute, qiIconMuteSelf, qiIconMuteServer, qiIconDeafSelf, qiIconDeafServer, qiIconMuteSuppressed;
```

Comment:
These should have the `qp` prefix is they're pixmaps. That, or refactor to use the `m_` prefix.

Reply:
Right.